kalamine/cli.py

```diff
@@ -114,15 +114,22 @@ def file_creation_context(ext: str = "") -> Iterator[Path]:
     default=False,
     help="Apply Angle-Mod (which is a [ZXCVB] permutation with the LSGT key (a.k.a. ISO key))",
 )
+@click.option(
+    "--qwerty-shortcut",
```

Comment:
--qwerty-shortcut**s** please (trailing `s`)
(I’m sorry to be that kind of guy)

Reply:
Don’t be, you’re right about it :)